components/editor/modules/meta/ui.js

```diff
@@ -0,0 +1,166 @@
+import React from 'react'
+import { css } from 'glamor'
+import { Map, Set } from 'immutable'
+
+import { Interaction, Field, Label, colors } from '@project-r/styleguide'
+import AutosizeInput from 'react-textarea-autosize'
+
+import withT from '../../../../lib/withT'
+import ImageInput from './ImageInput'
+import FBPreview from './FBPreview'
+import TwitterPreview from './TwitterPreview'
+
+const GUTTER = 20
+const styles = {
+  grid: css({
+    clear: 'both',
+    width: `calc(100% + ${GUTTER}px)`,
+    margin: `0 -${GUTTER / 2}px`,
+    ':after': {
+      content: '""',
+      display: 'table',
+      clear: 'both'
+    }
+  }),
+  span: css({
+    float: 'left',
+    paddingLeft: `${GUTTER / 2}px`,
+    paddingRight: `${GUTTER / 2}px`,
+    minHeight: 1,
+    width: '50%'
+  }),
+  container: css({
+    marginTop: 100,
+    backgroundColor: colors.secondaryBg,
+    padding: 30
+  }),
+  center: css({
+    maxWidth: 640,
+    margin: '0 auto'
+  }),
+  autoSize: css({
+    minHeight: 40,
+    paddingTop: '7px !important',
+    paddingBottom: '6px !important'
+  })
+}
+
+const Form = ({
+  t,
+  onInputChange,
+  data,
+  getWidth = key => key.match(/title/i) ? '100%' : ''
```

Comment:
I know you love inlining, but I would move that out into a separate
const getWidth = key => key.match(/title/i) ? '100%' : ''